common/transport/mqtt/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "azure-iot-mqtt-base",
-  "version": "1.12.2",
+  "version": "1.12.4",
```

Comment:
How come this version jump up two?

Reply:
My bad.  Not harmful

---
In reply to: [450513374](https://github.com/Azure/azure-iot-sdk-node/pull/838#discussion_r450513374) [](ancestors = 450513374)